src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -897,23 +922,27 @@ void readEvent(ReaderGroupManager readerGroupManager,String evtReaderGroupName,
 		evtOp.finishResponse();
 		val evtRead = evtRead_;
 		val evtData = evtRead.getEvent();
+		if (tailReadFlag) {
+			val timestampBuffer = ByteBuffer.allocate(TIMESTAMP_LENGTH);
+			timestampBuffer.put(evtData.array(),0,TIMESTAMP_LENGTH);
```

Comment:
A really minor notice: this comma separation style is differ from other code - it misses white space. I'd consider to add it.

Reply:
+